pkg/exchange/binance/parse.go

```diff
@@ -17,8 +18,8 @@ import (
 )
 
 type EventBase struct {
-	Event string `json:"e"` // event name
-	Time  int64  `json:"E"` // event time
+	Event string      `json:"e"` // event name
+	Time  json.Number `json:"E"` // event time
```

Comment:
you can use type MillisecondTimestamp here instead of this json.Number, it can unmarshal the timestamp

Reply:
done